src/mono/mono/mini/cpu-amd64.md

```diff
@@ -24,9 +24,9 @@
 #
```

Comment:
This file doesn't seem to have any markdown syntax. I think it should be a txt file.

Reply:
I would exclude them for this PR and deal with these files separately if necessary